src/ScoreDisabling.cpp

```diff
@@ -0,0 +1,90 @@
+#include "ScoreDisabling.hpp"
+#include "logger.h"
+
+namespace LeaderboardCore::Scoring {
+    std::unordered_map<std::string, std::unordered_map<ModInfo, std::string>> ScoreDisabling::disablingMods;
+    void ScoreDisabling::Disable(const ModInfo& modInfo, const std::string_view& reason) {
+        for (auto& [leaderboardId, disablingModInfos] : disablingMods) {
```

Comment:
What's the reason for `auto&` and not `const auto&`?

Reply:
I insert into the given disablingModInfos map, so it can't be const since I change it

Reply:
![image](https://user-images.githubusercontent.com/51795403/204375950-50bf387d-92f4-44ca-b139-8d6f2533d0dc.png)

Reply:
alright